terraform/setup/main.tf

```diff
@@ -12,13 +12,18 @@ resource "aws_dynamodb_table" "performance-dynamodb-table" {
   read_capacity  = 10
   write_capacity = 10
   hash_key       = "Service"
-  range_key      = "CommitHash"
+  range_key      = "UniqueID"
 
   attribute {
     name = "Service"
     type = "S"
   }
 
+  attribute {
```

Comment:
Adding a range key to provide a uniqueness for every row in database without conflicting to each other (since we have too many use case and a single commit hash won't cover enough for our use case)